src/System.Devices.Gpio/System.Devices.Gpio.Samples/Program.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Diagnostics;
+using System.Threading;
+
+namespace System.Devices.Gpio.Samples
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
```

Comment:
That project is just for me to test the API. It is not intended to be a console sample for users yet. Make sense to have samples, but since this code will change quite a lot in the future to follow API changes, I think is not worth to invest too much time on it for the moment.